partner_contact_in_several_companies/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
aren't we dropping utf8 headers? Let's take a decision :smile: I have some pending PRs where I've already removed them and some to be updated... Just want to make sure we are doing the same thing for all the PRs/modules

Reply:
on all .py files ?